projects/wallet-template/src/containers/WalletPopup/pages/Accounts/Accounts.tsx

```diff
@@ -110,10 +111,24 @@ const AccountsList: React.FC<AccountsListProps> = ({ keysets }) => {
 
 export const Accounts = () => {
   const navigate = useNavigate()
-  const { data: keysets } = useSWR("/rpc/keysets", async () => {
-    return rpc.client.listKeysets()
+  const location = useLocation()
+  const [fetchId, setFetchId] = useState(ulid())
```

Comment:
Why do we need a `fetchId`?

Reply:
i need a way for the account data to re-fetch when the page is reloaded or navigated to

Reply:
there are a few options in `useSWR` that serve for that purpose

https://swr.vercel.app/docs/revalidation